modular_skyrat/modules/titlescreen/code/titlescreen_html.dm

```diff
@@ -0,0 +1,101 @@
+
+GLOBAL_LIST_EMPTY(startup_messages)
+// FOR MOR INFO ON HTML CUSTOMISATION, SEE: https://github.com/Skyrat-SS13/Skyrat-tg/pull/4783
+
+#define MAX_STARTUP_MESSAGES 27
+
+/mob/dead/new_player/proc/get_lobby_html()
+	var/dat = GLOB.lobby_html
+	if(SSticker.current_state == GAME_STATE_STARTUP)
+		dat += {"<img src="titlescreen.gif" class="fone" alt="">"}
+		dat += {"
+		<div class="container_terminal">
+			<p class="menu_b">SYSTEMS INITIALIZING:</p>
+		"}
+		var/loop_index = 0
+		for(var/i in GLOB.startup_messages)
+			if(loop_index >= MAX_STARTUP_MESSAGES)
+				break
+			dat += i
+			loop_index++
+		dat += "</div>"
+
+	else
+		dat += {"<img src="titlescreen.gif" class="fone" alt="">"}
```

Comment:
Why is this between curly brackets, but sometimes it'd done without curly brackets?

Reply:
It's pure HTML code, without, it does not work